apps/src/codebridge/codebridgeContext/codebridgeContextProjectReducer.ts

```diff
@@ -45,7 +47,10 @@ export const projectReducer = (project: ProjectType, action: ReducerAction) => {
         folderId,
       };
 
-      return newProject;
+      return projectReducer(newProject, {
+        type: PROJECT_REDUCER_ACTIONS.ACTIVATE_FILE,
+        payload: {fileId},
+      });
```

Comment:
This prevents us from needing to call `setActiveFile` in the `newFile` function, which was causing issues with a double dispatch.

It's a _smidgen_ bit dangerous doing it this way, since logic may be introduced in the `setActiveFile` function that would need to be duplicated here, but we're probably okay.